packages/data/README.md

```diff
@@ -1028,6 +1028,52 @@ function Component() {
 }
 ```
 
+### Normalizing Selector Arguments
```

Comment:
Maybe the best approach is to briefly explain that each resolver has its resolution status cached in an internal state where the key is composed of the arguments supplied to the selector at _call time_.

This cache is used to determine the resolution status of a given resolver and helps to avoid potential additional calls to resolvers (which often undertake "expensive" operations such as network requests).

As a result it's important that arguments remain _consistent_ when _calling_ the selector. For example these two calls will not be cached using the same key, even though they are likely identical:

```js
// Arg as number
getSomeDataById( 123 );

// Arg as string
getSomeDataById( '123' );
```

To help with this selectors can optionally define a `normalizeArgs` property which should be a function which accepts the arguments passed to the selector and returns a normalized form of those same arguments.

...

Reply:
Ok I updated things with:

- better description
- code references